src/pages/EthBridge/WalletBalances.tsx

```diff
@@ -10,30 +9,9 @@ import { AuthWarning } from '../../components/AuthWarning';
 import Loader from 'react-loader-spinner';
 import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
-import UnlockToken from 'components/Earn/EarnRow/UnlockToken';
-import { messages, messageToString } from './messages';
 import { createNotification } from '../Exchange/utils';
-import { NETWORKS } from './index';
+import Flip from 'react-reveal/Flip';
```

Comment:
AFAIK react-reveal isn't maintained and doesn't support react 17+. I would recommend using something like react-awesome-reveal or a different package